src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -740,12 +721,6 @@ namespace winrt::TerminalApp::implementation
 
             switch (changedEvent)
             {
-            case CollectionChange::ItemChanged:
```

Comment:
huh, why is this one not necessary anymore?

Reply:
So I've never actually managed to get `ItemChanged` to fire when playing around with this. I'm pretty sure it fires when the observed vector has a `SetAt` performed on it. Since our Tab operations are Add/Remove/Reorder (which is also a remove + add), `ItemRemoved` and `ItemInserted` cover all of our cases. I felt like the `ItemChanged` case was unnecessary.

Although, in light of your next comment about covering enum values, it would be safer to have `ItemChanged` in case I'm totally wrong and missed a whole use case or future use case. 👍